components/raftstore/src/store/snap.rs

```diff
@@ -2099,18 +2101,31 @@ impl TabletSnapManager {
         PathBuf::from(&self.base).join(prefix)
     }
 
-    pub fn delete_snapshot(&self, key: &TabletSnapKey) -> bool {
-        let path = self.tablet_gen_path(key);
-        if path.exists() && let Err(e) = file_system::trash_dir_all(&path) {
-            error!(
-                "delete snapshot failed";
-                "path" => %path.display(),
-                "err" => ?e,
-            );
-            false
-        } else {
-            true
+    pub fn delete_snapshot(&self, region_id: u64, to_peer: Option<u64>) -> Result<()> {
```

Comment:
Doesn't need to change.

Reply:
What's your means that the parameter also using `TabletSnapKey`?  But it will skip if the concurrency is more than the limit.

Reply:
The key can still be built in this case.